tests/Diagnostics.HealthChecks.UnitTests/HealthChecksBuilderExtensionsTests.cs

```diff
@@ -25,6 +25,7 @@ public class HealthChecksBuilderExtensionsTests
 	{
 		[DataTestMethod]
 		[DynamicData(nameof(GetHeaders), DynamicDataSourceType.Method)]
+		[Obsolete("The health check implementation based on AbstractHealthCheck is obsolete.")]
```

Comment:
Why are the unit test methods being marked as obsolete?

Reply:
elsewhere too

Reply:
It wouldn't compile otherwise because the projects are set to treat warnings directly as errors.
Another possibility would be to ignore that kind of warnings, but in that case we would risk not getting other obsolete warnings from other packages.